dask_planner/src/expression.rs

```diff
@@ -653,6 +653,30 @@ impl PyExpr {
             ))),
         }
     }
+
+    /// Returns if a sort expressions is an ascending sort
+    #[pyo3(name = "isSortAscending")]
+    pub fn is_sort_ascending(&self) -> PyResult<bool> {
+        match &self.expr {
+            Expr::Sort { asc, .. } => Ok(asc.clone()),
```

Comment:
[Docs](https://docs.rs/datafusion/latest/datafusion/logical_plan/enum.Expr.html#variant.Sort) show `asc` as a Rust Scalar type. Could use `*asc` to avoid an extra copy.